.github/workflows/matrix.yaml

```diff
@@ -7,6 +7,11 @@ on:
         description: "The weaviate docker image to version to run tests against"
         type: string
         required: true
+      weaviate_max_major_version_test_set:
```

Comment:
Why not using a [choice](https://github.blog/changelog/2021-11-10-github-actions-input-types-for-manual-workflows/) option?
´´´
choice:
        type: choice
        description: Make a choice
        options:
        - foo
        - bar
        - baz
´´´
This way we reduce the faulty jobs launched because it was passed a wrong version

Reply:
don't have hard opinions, but my thought was to decrease any addition when we release a new version